src/libData/BlockData/Block/TxBlock.cpp

```diff
@@ -28,7 +28,8 @@ unsigned int TxBlock::Serialize(vector<unsigned char> & dst, unsigned int offset
 
     unsigned int header_size_needed = sizeof(uint8_t) + sizeof(uint32_t) + UINT256_SIZE + UINT256_SIZE + 
                                       BLOCK_HASH_SIZE + UINT256_SIZE + UINT256_SIZE + TRAN_HASH_SIZE + 
-                                      sizeof(uint32_t) + sizeof(uint32_t) + PUB_KEY_SIZE + UINT256_SIZE + BLOCK_HASH_SIZE;
+                                      TRAN_HASH_SIZE + sizeof(uint32_t) + sizeof(uint32_t) + PUB_KEY_SIZE + 
+                                      UINT256_SIZE + BLOCK_HASH_SIZE;
```

Comment:
:bulb: This expression appears several times. Consider extracting a constant.

Reply:
Yeah, even I was wondering the same when making the change.